### PR TITLE
coverage for unverified secrets

### PR DESCRIPTION
**Pull request by ErebusZ**
Added coverage for the unverified secrets detected by TruffleHog.
Now verified secrets will have a risk rating of **HIGH** and the unverified a risk rating of **POTENTIALLY**.

### CODE REVIEW

File: agent/trufflehog_agent.py [hunk 1]
```diff
@@ -34,11 +35,15 @@ class TruffleHogAgent(
     """
 
     def _report_vulnz(self, vulnz: list[dict[str, Any]], message: m.Message) -> None:
+        if len(vulnz) > 0:
+            logger.info("Reporting vulnerabilities.")
```
Comment (amine3) [inline agent/trufflehog_agent.py:39]: This is not needed.

File: tests/trufflehog_test.py [hunk 2]
```diff
@@ -56,11 +56,10 @@ def testSubprocessParameter_whenProcessingFile_beValid(
     trufflehog_agent_file.process(scan_message_file)
     args = subprocess_check_output_mock.call_args[0][0]
 
-    assert len(args) == 5
+    assert len(args) == 4
```
Comment (amine3) [inline tests/trufflehog_test.py:59]: Missing unit test to diff between verified and non verified.

**Review by amine3 — DISMISSED** [review]

File: tests/trufflehog_test.py (hunk 2 above)
Comment (ErebusZ) [reply tests/trufflehog_test.py:59]: [12e5213](https://github.com/Ostorlab/agent_trufflehog/pull/5/commits/12e5213d50793f97575d8917d4acd433bdb27454) Added.

File: agent/trufflehog_agent.py (hunk 1 above)
Comment (ErebusZ) [reply agent/trufflehog_agent.py]: removed.

**Pull request merged**